tests/test_SE_combine_methods.py

```diff
@@ -49,8 +48,8 @@ def test_SE_combineCols_useNames_false(summarized_experiments):
         combined=combined,
         shape=(3, 6),
         assay_names=["counts", "lognorm"],
-        rownames=["HER2", "BRCA1", "TPFK"],
-        rowData_cols=["seqnames", "start", "end"],
+        rownames=[i for i in range(3)],
+        rowData_cols=["seqnames", "start", "end", "index"] * 2,
         colnames=["cell_1", "cell_2", "cell_3", "cell_4", "cell_5", "cell_6"],
         colData_cols=["sample", "disease", "doublet_score"],
     )
```

Comment:
`combineCols()` should preserve the same rownames of the first `SummarizedExperiment`

Reply:
fixed this in the new set of commits